lib/utilities/monitoringHandler.js

```diff
@@ -3,65 +3,65 @@ const client = require('prom-client');
 
 const collectDefaultMetrics = client.collectDefaultMetrics;
 const numberOfBuckets = new client.Gauge({
-    name: 'cloud_server_number_of_buckets',
+    name: 's3_cloudserver_buckets',
     help: 'Total number of buckets',
 });
 const numberOfObjects = new client.Gauge({
-    name: 'cloud_server_number_of_objects',
+    name: 's3_cloudserver_objects',
     help: 'Total number of objects',
 });
 const lastReportTimestamp = new client.Gauge({
-    name: 'cloud_server_last_report_timestamp',
+    name: 's3_cloudserver_last_report_timestamp',
     help: 'Timestamp of last object/bucket count report',
 });
 const numberOfIngestedObjects = new client.Gauge({
-    name: 'cloud_server_number_of_ingested_objects',
-    help: 'Number of out of band ingestion',
+    name: 's3_cloudserver_ingested_objects',
```

Comment:
should have a unit : `..._ingested_objects_count`

Reply:
same as above.